.github/workflows/labeler.yml

```
@@ -0,0 +1,19 @@
+# Add labels to pull request
+# uses logic in .github/labeler.yml
+
+---
+name: "Add labels to pull request"
+
+on:
+- pull_request_target
```

```suggestion
  pull_request_target:
    branches: [ 'master' ]
```

so that backport PRs (manual or patchback) don't trigger labelling